nilearn/decoding/space_net.py

```diff
@@ -326,6 +326,27 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
 
     solver_params: dict
        Dictionary of param-value pairs to be passed to solver.
+
+    is_classif: ??? TODO: Add description.
```

Comment:
Bool, 
Indicates whether the loss is a classification loss or a regression loss